ui/admin/app/controllers/scopes/scope/auth-methods/index.js

```diff
@@ -4,8 +4,11 @@
  */
 
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
 
 export default class ScopesScopeAuthMethodsIndexController extends Controller {
+  // =services
+
```

Comment:
non-blocking: Why was this added?

Reply:
oh that's not necessary (at least for now), ill remove it.